.compatibility_tests/compatibility_test_1_18/Cargo.toml

```diff
@@ -29,13 +29,13 @@ struqture_1_9 = { git = "https://github.com/HQSquantumsimulations/struqture", pa
 bincode = { version = "1.3" }
 ndarray = "0.16"
 qoqo_calculator = { git = "https://github.com/HQSquantumsimulations/qoqo_calculator" }
-qoqo_calculator_1_2 = { git = "https://github.com/HQSquantumsimulations/qoqo_calculator", package = "qoqo_calculator", tag = "v1.2.4" }
+qoqo_calculator_1_2 = { git = "https://github.com/HQSquantumsimulations/qoqo_calculator", package = "qoqo_calculator", tag = "v1.4.4" }
```

Comment:
Also here, is this the correct version? If it is, I'd suggest renaming qoqo_calculator_1_2